### DISCUSSION TITLE
Help with some questions about the ZBL form

### DISCUSSION THREAD

**Discussion by manbujin** [deepmodeling/deepmd-kit#2130] [Q&A] [answered]
Hello！
    I would like to ask you in what order I should prepare the ZBL table if the system I want to train has four elements, and the order in POSCAR is ABCD.
   Is it in the order of A-A、A-B、A-C、A-D、B-B、B-C、B-D、C-C、C-D、D-D？
Thank you so much！

**Comment by wanghan-iapcm** [comment, marked as answer]
Suppose that A is type 0, B is 1, C is 2 and D is 3, then the order of columns in the tabulation file is 
 A-A、A-B、A-C、A-D、B-B、B-C、B-D、C-C、C-D、D-D

**Reply by manbujin** [reply]
Thank you very much for your answer
